articles/openshift/howto-using-azure-redhat-openshift.md

```diff
@@ -39,25 +39,25 @@ The `az aro` extension allows you to create, access, and delete Azure Red Hat Op
 > The `az aro` extension is currenty in preview. It may be changed or removed in a future release.
 > To opt-in for the `az aro` extension preview you need to register the `Microsoft.RedHatOpenShift` resource provider.
 > 
->    ```console
+>    ```azurecli
 >    az provider register -n Microsoft.RedHatOpenShift --wait
 >    ```
 
 1. Log in to Azure.
 
-   ```console
+   ```azurecli
    az login
    ```
 
 2. Run the following command to install the `az aro` extension:
 
-   ```console
+   ```azurecli
    az extension add -n aro --index https://az.aroapp.io/preview
```

Comment:
The command is now available directly as a `az aro` without need for instalation. It has been release in latest version https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/docs-ref-conceptual/release-notes-azure-cli.md#aro

Reply:
I removed the bullet and prereq section.